Installation fails under Linux Mint 19, kernel 4.15

Hello,

First of all, many thanks for all your work, much appreciated.

Now, I'm trying to install the drivers but I'm stopped at this step:

```
CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/rtw_cfgvendor.o
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/wifi_regd.o
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/rtw_android.o
/var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_cfg80211.c: In function ‘rtw_cfg80211_indicate_sta_assoc’:
/var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_cfg80211.c:4484:3: error: implicit declaration of function ‘cfg80211_sinfo_alloc_tid_stats’; did you mean ‘cfg80211_inform_bss_width’? [-Werror=implicit-function-declaration]
   cfg80211_sinfo_alloc_tid_stats(&sinfo, GFP_KERNEL);
   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   cfg80211_inform_bss_width
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/rtw_rhashtable.o
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_mp.o
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/hal/hal_intf.o
  CC [M]  /var/lib/dkms/rtl8812au/5.6.4.1/build/hal/hal_com.o
/var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_cfg80211.c: At top level:
cc1: warning: unrecognized command line option ‘-Wno-cast-function-type’
cc1: warning: unrecognized command line option ‘-Wno-cast-function-type’
cc1: some warnings being treated as errors
scripts/Makefile.build:332: recipe for target '/var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_cfg80211.o' failed
make[2]: *** [/var/lib/dkms/rtl8812au/5.6.4.1/build/os_dep/linux/ioctl_cfg80211.o] Error 1
make[2]: *** Waiting for unfinished jobs....
Makefile:1552: recipe for target '_module_/var/lib/dkms/rtl8812au/5.6.4.1/build' failed
make[1]: *** [_module_/var/lib/dkms/rtl8812au/5.6.4.1/build] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.15.0-20-generic'
Makefile:1606: recipe for target 'modules' failed
make: *** [modules] Error 2
```

Linux Mint 19 4.15.0-20-generic #21-Ubuntu SMP x86_64 x86_64 x86_64 GNU/Linux

Any idea about this?
Thanks

--- a/.travis.yml
+++ b/.travis.yml
@@ -74,3 +74,13 @@ matrix:
             - gcc-7
             - libssl1.1
       env: COMPILER=gcc-7 KVER=4.19.67
+    - compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+            - sourceline: 'ppa:ondrej/nginx-mainline'
+          packages:
+            - gcc-7
+            - libssl1.1
+      env: COMPILER=gcc-7 KVER=4.15.18

--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -4481,7 +4481,9 @@ void rtw_cfg80211_indicate_sta_assoc(_adapter *padapter, u8 *pmgmt_frame, uint f
 		sinfo.filled = STATION_INFO_ASSOC_REQ_IES;
 		sinfo.assoc_req_ies = pmgmt_frame + WLAN_HDR_A3_LEN + ie_offset;
 		sinfo.assoc_req_ies_len = frame_len - WLAN_HDR_A3_LEN - ie_offset;
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4, 18, 0))
 		cfg80211_sinfo_alloc_tid_stats(&sinfo, GFP_KERNEL);
+#endif
 		cfg80211_new_sta(ndev, get_addr2_ptr(pmgmt_frame), &sinfo, GFP_ATOMIC);
 	}
 #else /* defined(RTW_USE_CFG80211_STA_EVENT) */

